benches/arrays.rs

```diff
@@ -2,6 +2,24 @@ use criterion::{criterion_main, Criterion};
 use nickel_lang_utilities::{ncl_bench_group, EvalMode};
 use pprof::criterion::{Output, PProfProfiler};
 
+/// Generates a pseaudo-random Nickel array as a string.
+fn ncl_random_array(len: usize) -> String {
+    let m = 2_u64.pow(32);
+    let a = 1664525;
+    let c = 1013904223;
+
+    let mut numbers = Vec::with_capacity(len);
+    numbers.push(1337);
+
+    for _ in 0..len {
+        let x = *numbers.last().unwrap();
+        numbers.push((a * x + c) % m);
+    }
+
+    // HACK: It so happens that this is valid Nickel syntax.
+    format!("{:?}", numbers)
```

Comment:
May it be more robust to instantiate an `Term::Array` and use the ast pretty-printing facilities, so that if the syntax change, this won't break? Honestly I don't think the array syntax will ever become invalid... but, if it's only a few line...